code/obj/item/clothing/masks.dm

```diff
@@ -130,6 +130,25 @@
 		..()
 		setProperty("disorient_resist_eye", 20)
 
+/obj/item/clothing/mask/gas/respirator
+	name = "gas respirator"
+	desc = "A close-fitting gas mask with a custom particle filter."
+	icon_state = "respirator"
+	item_state = "respirator"
+	uses_multiple_icon_states = 1
+	color_r = 0.9
+	color_g = 0.9
+	color_b = 0.95 // sunglasses config
```

Comment:
Shouldn't this tint blue? Maybe copy whatever tint the NTSO gas mask is using?

Reply:
sunglasses tint slightly blue by default, though i've messed with the numbers a bit to bring them more in line with NTSO